r/R/dplyr-functions.R

```diff
@@ -808,3 +808,25 @@ agg_funcs$all <- function(x, na.rm = FALSE) {
     options = list(na.rm = na.rm, na.min_count = 0L)
   )
 }
+
+agg_funcs$mean <- function(x, na.rm = FALSE) {
+  list(
+    fun = "mean",
+    data = x,
+    options = list(na.rm = na.rm, na.min_count = 0L)
+  )
+}
+agg_funcs$sd <- function(x, na.rm = FALSE) {
+  list(
+    fun = "stddev",
+    data = x,
+    options = list(ddof = 1)
```

Comment:
I've manually set `ddof` here, similarly to how we've manually set `na.min_count` without exposing this to the end user, but actually, would it make sense to add it as an argument with a default value of `1`?  I could see `ddof` (i.e. delta degrees of freedom) maybe being useful in some statistical contexts.

What governs whether we just mimic existing R behaviour vs. expose Arrow's additional functionality?

Reply:
For scalar functions, you can always call the `arrow_` version of the functions if you want arrow behaviors, but that won't work here. Maybe add it to the signature of `agg_funcs$sd` so it can be passed in?